File: Android/sources/com/android/billingclient/api/zzy.java
package com.android.billingclient.api;

import java.util.List;
import java.util.concurrent.Callable;

public final /* synthetic */ class zzy implements Callable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ String zzb;
    public final /* synthetic */ List zzc;
    public final /* synthetic */ SkuDetailsResponseListener zzd;

    public /* synthetic */ zzy(BillingClientImpl billingClientImpl, String str, List list, String str2, SkuDetailsResponseListener skuDetailsResponseListener) {
        this.zza = billingClientImpl;
        this.zzb = str;
        this.zzc = list;
        this.zzd = skuDetailsResponseListener;
    }

    public final Object call() {
        this.zza.zzo(this.zzb, this.zzc, (String) null, this.zzd);
        return null;
    }
}

File: IOS/v2/com/android/billingclient/api/zzy.swift
package com.iOS.billingclient.api;.swift
import java.util.List;.swift
import java.util.concurrent.Callable;.swift
public final /* synthetic */ class zzy implements Callable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ String zzb;.swift
public final /* synthetic */ List zzc;.swift
public final /* synthetic */ SkuDetailsResponseListener zzd;.swift
public /* synthetic */ zzy(BillingClientImpl billingClientImpl, String str, List list, String str2, SkuDetailsResponseListener skuDetailsResponseListener) {.swift
this.zza = billingClientImpl;.swift
this.zzb = str;.swift
this.zzc = list;.swift
this.zzd = skuDetailsResponseListener;.swift
}.swift
public final Object call() {.swift
this.zza.zzo(this.zzb, this.zzc, (String) null, this.zzd);.swift
return null;.swift
}.swift
}.swift
